### ISSUE TITLE
Send warning like messages to users but don't add a strike

### ISSUE THREAD

**Issue by tchristiansen** [mastodon/mastodon#22807]
### Solution
Add "Contact" option/feature to the moderation view:
![Bildschirm­foto 2022-12-28 um 16 15 43](https://user-images.githubusercontent.com/4445885/209836777-13b1eb4a-eb52-4ea1-bad5-c17b664a27fb.png)
With the option to send messages to the user without putting a strike on them:
![Bildschirm­foto 2022-12-28 um 16 24 24](https://user-images.githubusercontent.com/4445885/209836787-301176b8-225d-46d4-a756-7abaef1ec10b.png)

### Optional
Add a checkbox to disable strike on user when sending a warning.

### Motivation

We have some preset welcome mails, kind reminders to out server rules etc. which we send to our users. As not all moderators should have access to out mailbox we would like to have an option to send those mails directly from the moderation system.

**Comment by ThisIsMissEm** [comment]
This is perhaps covered by #31081

**Comment by leahoswald** [comment]
I would really love to see such an option. We regularly use the warning feature to inform users about behavior we think that could be better but is not worth a formal warning. Currently the "Send a warning" feature is the only possible way to do so from the moderation interface.  Sadly we regularly see users that are very unsettled or even anxious after receiving such a "Warning" despite the fact we just want to tell them "hey, no immediate problem, do better next time". So it would be great to have another message option with a more neutral template instead of adding something like this before every "warning":

> First off: please don't be alarmed. Mastodon only allows us to send “warnings” as a moderation action when we want to contact people. Please read this message more like a note or a request rather than a menacing capital-W Warning. If you have questions or comments about it, please contact us at <mod account> at any time.

**Comment by Tealk** [comment]
But then there also needs to be some kind of chat interface in the admin area, or should the user not be able to reply?

**Comment by leahoswald** [comment]
Nope, a reply function is not really needed. They could use the other contact methods if they need to. Or it could be integrated in the dispute interface from warnings. What I have in mind is the same as Warning but without calling it warning and another mail template.

**Comment by leahoswald** [comment]
It isn't a warning. Its an informational message. Not giving it the warning vibe is exactly what I want. And yes direct communication would be even better, but currently warnings are the only option in the moderation UI to inform a user via mail and I think since 4.2 also via a notification without doing this manually twice. And it also keeps track of that communication which isn't given with the manual approach but is important for further moderation in the future and also if you are not a single mod.  It is an easy approach that would cost not that much time to implement but a whole bidirectional communication system like @ThisIsMissEm suggested would be even better but is much more work and to be fair I don't think this will come some.
